src/status_im/contexts/chat/messenger/messages/content/view.cljs

```diff
@@ -126,14 +126,62 @@
       :photo-path          photo-path
       :incoming?           (not= public-key from)}]))
 
+(defn group-member
+  [literal]
+  (let [[primary-name _] (rf/sub [:contacts/contact-two-names-by-identity literal])
+        photo-path       (rf/sub [:chats/photo-path literal])]
+    [rn/view {:style style/group-member-mention}
+     [quo/user-avatar
+      {:full-name         primary-name
+       :profile-picture   photo-path
+       :status-indicator? false
+       :size              :xxxs
+       :ring?             false}]
+     [quo/text
+      {:weight :semi-bold
+       :style  {:margin-left 4}}
+      primary-name]]))
+
+
+(defn resolve-group-system-message
+  [{:keys [^js children]} acc]
+  (conj acc
+        (reduce (fn [acc {:keys [type literal]}]
+                  (if (= type "mention")
+                    (conj acc [group-member literal])
+                    (conj acc
+                          [:<>
```

Comment:
Looks like we don't need this wrapping `:<>`.

Reply:
there is probably a better way to achieve this with `flex-box` but without `:<>` the text is a little mis-aligned if you can notice it in the screenshot here

<img width="291" alt="Screenshot 2024-07-25 at 14 26 49" src="https://github.com/user-attachments/assets/e0a3ed06-f6bf-4fd9-9eb8-f588f79f9d60">

Reply:
actually your are right its not.